content/docs/tutorials/sourcegraph.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Sourcegraph
+---
+
+*This tutorial provides step-by-step instructions on how to rotate a Sourcegraph Access Token.*
+
+---
+
+## Generate a New Sourcegraph Access Token
+
+#### Step 1a. Navigate to the Access Token page
+Navigate to the url [https://sourcegraph.com/users/{username}/settings/tokens](https://sourcegraph.com/users/{sourcegraph_username}/settings/tokens). Replace `{username}` with your Sourcegraph username. If this token is from different Sourcegraph instance, replace the url with the instance url example: https://sourcegraph.example.com/users/{username}/settings/tokens.
+
+![Access token page](/images/sourcegraph/1.png)
+
+#### Step 1b. Generate Access Token
+Type in a new description for the Access Token.
+
+#### Step 1c. Confirm
+Click the `Generate Token` button to confirm the operation.
+
+![Regenerate access token](/images/sourcegraph/3.png)
+
+## Replace the Leaked Sourcegraph Access token
+Replace the leaked Sourcegraph access token with the new one in all impacted applications and services.
+
+### Revoke the Access Token
```

Comment:
Can you update this to be at heading level 2 to match the generate and replace step?